Pipfile

```diff
@@ -8,7 +8,7 @@ arxiv-base = "*"
 flask = "*"
 requests = "*"
 arxiv-auth = "==0.1.0rc6"
-"boto3" = "*"
+"boto3" = "==1.7.84"
```

Comment:
I pinned this, because moto has had some trouble keeping up with boto3 recently.